### PR TITLE
++ basic night mode toggle

### PR DESCRIPTION
**Pull request by cluzier**
### Description

Added a button to toggle night mode colors, added a js function to change body background and text colors

### Motivation and Context

A night mode was requested in the following issue:

https://github.com/dogecoin/dogecoin.com/issues/226

### How Has This Been Tested?

Using a localhost on an Apache server

Testing was done a local machine in a controlled environment

Changes only affect the body background color and text colors

### Screenshots (if appropriate):
image

### Checklist:

- [x] GitHub issue linked
- [ ] The necessary translations have been added for all languages
- [ ] Any documentation has been updated accordingly
- [x] Responsive sizes (Mobile) tested
- [x] Cross Browser tested if necessary
- [x] Conflicts resolved

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,6 +56,7 @@ <h1 class="sr-only">Dogecoin</h1>
                     <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#about"><span data-i18n="dgc-what-is-doge">What is dogecoin</span></a></li>
                     <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#start"><span data-i18n="dgc-get-started-now">How to get started</span></a></li>
                     <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded" href="guide.html"><span data-i18n="dgc-guide">Guide</span></a></li>
+                    <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded" href="#" id="toggleBtn" onclick="toggleDN()">🌙</a></li>
```
Comment (nickatnight) [inline index.html:59]: What about guide.html and support.html?

Comment (cluzier) [reply index.html:59]: my apologies, will add this tonight and push up changes.

**Comment by cluzier** [comment]
I've gone ahead and added it to the support and guide pages @nickatnight

**Review by Dogeyyboi — COMMENTED** [review]
I got something for yall



Screenshot 2021-05-27 8 38 20 AM

**Comment by Dogeyyboi** [comment]
try changing the document.body.style.color

File: index.html (hunk 1 above)
Comment (Dogeyyboi) [reply index.html:59]:
```suggestion
                    document.body.style.color = "black";
```

```suggestion
                    <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded" href="#" id="toggleBtn" onclick="toggleDN()">🌙</a></li>
```
Try that
 hope it helps

**Pull request closed without merging**